### PR TITLE
Enable global pooling ops

### PR DESCRIPTION
**Pull request by tsocha**
- Enable global pooling tests
- Enable global pooling ops

### CODE REVIEW

File: ngraph_onnx/onnx_importer/utils/conv.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
     from ngraph_onnx.onnx_importer.model_wrappers import NodeWrapper
 
 
-def get_pads(onnx_node: 'NodeWrapper') -> Tuple[List[int], List[int]]:
+def get_pads(onnx_node: 'NodeWrapper', kernel_shape: 'List[int]'=None) -> Tuple[List[int], List[int]]:
```
Comment (arogowie-intel) [inline ngraph_onnx/onnx_importer/utils/conv.py]: Why using type annotation within function signature only here?

File: ngraph_onnx/onnx_importer/utils/conv.py [hunk 2]
```diff
@@ -101,15 +102,16 @@ def get_kernel_shape(onnx_node):  # type: (NodeWrapper) -> List[int]
     return kernel_shape
 
 
-def get_strides(onnx_node):  # type: (NodeWrapper) -> List[int]
+def get_strides(onnx_node, kernel_shape=None):  # type: (NodeWrapper, List[int]) -> List[int]
```
Comment (arogowie-intel) [inline ngraph_onnx/onnx_importer/utils/conv.py]: Let's use `Optional[type]` type annotation for function parameters which may be None.

File: ngraph_onnx/onnx_importer/utils/pool.py [hunk 3]
```diff
@@ -26,15 +26,14 @@
...
-from ngraph_onnx.onnx_importer.utils.decorators import refactoring_required
 
 if TYPE_CHECKING:
     from ngraph_onnx.onnx_importer.model_wrappers import NodeWrapper
 
 log = logging.getLogger(__file__)
 
 
-def get_op_type(onnx_node):  # type: (NodeWrapper) -> str
+def get_op_type(onnx_node):  # type: (NodeWrapper) -> Tuple[str, bool]
```
Comment (arogowie-intel) [inline ngraph_onnx/onnx_importer/utils/pool.py]: Update docstring for return value.

File: ngraph_onnx/onnx_importer/utils/conv.py [hunk 4]
```diff
@@ -51,7 +51,8 @@ def get_pads(onnx_node: 'NodeWrapper') -> Tuple[List[int], List[int]]:
     """
     auto_pad = onnx_node.get_attribute_value('auto_pad')
     pads = onnx_node.get_attribute_value('pads', ())  # Padding along each axis
-    kernel_shape = get_kernel_shape(onnx_node)
+    if(kernel_shape is None):
```
Comment (arogowie-intel) [inline ngraph_onnx/onnx_importer/utils/conv.py]: Why `kernel_shape` may be None if you always pass it some value in `make_pooling_op`? The same for `get_strides`. Maybe checking whether `kernel_shape is None` should be done before passing its value to these functions.

File: ngraph_onnx/onnx_importer/utils/conv.py (hunk 2 above)
Comment (tsocha) [reply ngraph_onnx/onnx_importer/utils/conv.py]: @postrational

File: ngraph_onnx/onnx_importer/utils/conv.py (hunk 1 above)
Comment (tsocha) [reply ngraph_onnx/onnx_importer/utils/conv.py]: @postrational

File: ngraph_onnx/onnx_importer/utils/pool.py (hunk 3 above)
Comment (tsocha) [reply ngraph_onnx/onnx_importer/utils/pool.py]: Updated

File: ngraph_onnx/onnx_importer/utils/conv.py (hunk 1 above)
Comment (postrational) [reply ngraph_onnx/onnx_importer/utils/conv.py]: Let's stick to one convention. We use comments everywhere else, so let's move this to comments here too.
At some point we may globally switch to inline typing.

File: ngraph_onnx/onnx_importer/utils/conv.py (hunk 2 above)
Comment (postrational) [reply ngraph_onnx/onnx_importer/utils/conv.py]: That's not necessary. Let's omit `Optional` if the default value is `None`, mypy infers the `Optional` itself.

File: ngraph_onnx/onnx_importer/utils/pool.py [hunk 5]
```diff
@@ -26,30 +26,31 @@
...
     """
     Parse ONNX pooling operation attributes and produce an ngraph compatible pool_params dict.
 
     :param onnx_node: wrapped ONNX node for a pooling operation op
-    :return: dict of pool_params for ng.pooling
+    :return: dict of pool_params for ng.pooling and if it is a global pooling op
```
Comment (arogowie-intel) [inline ngraph_onnx/onnx_importer/utils/pool.py]: Update return value to conform with what is stated in the type annotation: Tuple[str, bool]. There is no dict mentioned.

File: ngraph_onnx/onnx_importer/utils/pool.py [hunk 6]
```diff
@@ -72,30 +71,36 @@ def reduce_extra_dims(spatial_dims_count, param_shape, onnx_node):
     return param_shape
 
 
-def make_pooling_op(onnx_node, ng_inputs, custom_pool_params=None):
-    # type: (NodeWrapper, List[NgraphNode], Dict) -> NgraphNode
+def make_pooling_op(onnx_node, ng_inputs, is_global=False, custom_pool_params=None):
```
Comment (postrational) [inline ngraph_onnx/onnx_importer/utils/pool.py]: Please make function `make_global_pooling_op`
In that function calculate the `kernel_shape`
Pass `kernel_shape` to `make_pooling_op`.
Perhaps pass it in `custom_pool_params`. What are `custom_pool_params` used for currently? 
Please document `custom_pool_params` or remove.

**Pull request merged**